.github/workflows/release.yaml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
     release:
         name: Publish release
-        runs-on: ubuntu-20.04
+        runs-on: ubuntu-latest
```

Comment:
any reason not to just jump to `latest` here?  Our release process is super straightforward.

Reply:
That's the `latest LTS,` so it should be fine, as Ubuntu is not as frequently updated as others.